src/package.json

```diff
@@ -5,12 +5,12 @@
   "main": "fingerprint-auth",
   "nativescript": {
     "platforms": {
-      "android": "2.5.0",
-      "ios": "2.5.0"
+      "android": "3.0.0",
```

Comment:
We discussed this and wonder if webpack and travis will succeed when still targeting 2.x - I'll find out soon!